app/src/main/java/org/wikipedia/edit/EditSectionActivity.kt

```diff
@@ -494,13 +505,31 @@ class EditSectionActivity : BaseActivity(), ThemeChooserDialog.Callback {
      * based on the current state of the button.
      */
     fun clickNextButton() {
+        val addImageTitle = intent.getParcelableExtra<PageTitle>(InsertMediaActivity.EXTRA_IMAGE_TITLE)
+        val addImageSource = intent.getStringExtra(InsertMediaActivity.EXTRA_IMAGE_SOURCE)
         when {
             editSummaryFragment.isActive -> {
+                if (invokeSource == Constants.InvokeSource.EDIT_ADD_IMAGE) {
+                    ImageRecommendationsEvent.logAction("editsummary_save", "editsummary_dialog", ImageRecommendationsEvent.getActionDataString(
```

Comment:
Do you think it would be better if we put these log actions into `ImageRecommendationsEvent`?

For example: 
```
fun logEditSummaryAction(actionData: String = "", wikiId: String = "") {
        logAction("editsummary_save", "editsummary_dialog", actionData, wikiId)
}
```

Reply:
`action` string is the one that changes per event, so i feel this wont be useful.

Reply:
Got it. It makes sense to me.